File: Python/41A. Translation.py
s =  input()
t = input()
y = 0
for i in range(len(s)):
    if len(s) != len(t):
        break
    if s[i] == t[len(s)-1-i]:
        y += 1
if y == len(s):
    print("YES")
else:
    print("NO")

File: C++/41A. Translation.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    string s, t;
    int i, j, y = 0;
    cin>>s;
    cin>>t;
    for(i=0; i<s.size(); i++){
        if(s[i]==t[t.size()-1-i]){
            y++;
        }
    }
    if(y==s.size()){
        cout<<"YES";
    }
    else{
        cout<<"NO";
    }
    return 0;
}